### DISCUSSION TITLE
Some DITA-OT errors (like DOTJ046E) do not show up on GitHub Action in the log as expected

### DISCUSSION THREAD

**Discussion by stefan-jung** [dita-ot/dita-ot#4462] [Q&A] [answered]
Hi,

I'm publishing with GitHub Actions with non-verbose logging and capture the logging info with `tee`. This works fine.

I am using this command:

```yaml
        run: dita-ot/bin/dita --project=my-docs.xml --processing-mode=lax --store-type=file --parallel=true --conserve-memory=true -Ddita.preprocess.reloadstylesheet=false -Dskip.pdf=true -Dgenerate-debug-attributes=false --no-color | tee my-docs.log
```

However, some errors, e.g. `DOTJ046E` shows up in the build log on GitHub action. But it is not captured by the `tee`d log:

![image](https://github.com/dita-ot/dita-ot/assets/3266567/3586d491-6b7b-4193-a0d2-4da1e67c5731)

The line is simply missing in the log. I have no idea why. Do you have an idea?

**Comment by jelovirt** [comment, marked as answer]
`tee` will only redirect standard out. Try redirecting standard error into standard out and then use `tee` as before. So `… 2>&1 | tee …`

**Comment by chrispy-snps** [comment]
Some shells provide a pipe shortcut to include standard error (e.g. `|& tee`).
